tests/integration/bigquery.spec.ts

```diff
@@ -80,12 +80,19 @@ suite("@dataform/integration/bigquery", { parallel: true }, ({ before, after })
       // Drop all the tables before we do anything.
       await cleanWarehouse(compiledGraph, dbadapter);
 
-      // Drop the meta schema
-      await dbadapter.dropSchema("dataform-integration-tests", "dataform_meta");
-
       // Run the project.
       let executionGraph = await dfapi.build(compiledGraph, {}, dbadapter);
-      let executedGraph = await dfapi.run(dbadapter, executionGraph).result();
+      let runResult = await dfapi.run(dbadapter, executionGraph).result();
+      const previouslyExecutedActions = runResult.actions
```

Comment:
it would be nice if the API returned something that could be passed immediately back in to the next run, rather than having each caller need to re-implement something like this. Not sure what that is exactly, but could be possible?

Reply:
We can consider a new API going forward, but IMO the reality is that any caller who wants to use run caching is going to have to store some state _somewhere_, and so this code is going to look different for each such case. (e.g. Our closed source code doesn't look like this.)